packages/sfc-playground/vite.config.ts

```diff
@@ -4,7 +4,7 @@ import { defineConfig, Plugin } from 'vite'
 import vue from '@vitejs/plugin-vue'
 import execa from 'execa'
 
-const commit = execa.sync('git', ['rev-parse', 'HEAD']).stdout.slice(0, 7)
+const commit = execa.sync('git', ['rev-parse', '--short', 'HEAD']).stdout
```

Comment:
We can pass a specific length
```suggestion
const commit = execa.sync('git', ['rev-parse', '--short=7', 'HEAD']).stdout
```

If the 7-length one is not unique, it'll increase the length to make sure it's unique.